test/functional/api/vim_spec.lua

```diff
@@ -177,6 +178,62 @@ describe('vim_* functions', function()
     end)
   end)
 
+  describe('err_write', function()
+    before_each(function()
+      clear()
+      screen = Screen.new(40, 8)
+      screen:attach()
+      screen:set_default_attr_ids({
+        [1] = {foreground = Screen.colors.White, background = Screen.colors.Red},
+        [2] = {bold = true, foreground = Screen.colors.SeaGreen}
+      })
+      screen:set_default_attr_ignore( {{bold=true, foreground=Screen.colors.Blue}} )
+    end)
+
+    it('can show one line', function()
+      nvim_async('err_write', 'has bork\n')
```

Comment:
It would show up to `&cmdheight` lines, i.e two, then showing more will be blocked on pressing RETURN. Given the async nature, the behaviour could very well be different when nvim happens to be in some "global" processing mode, so that all are shown at once, but certainly a test could be added for the simple case (nvim is idle).